pkg/cninswrapper/ns.go

```diff
@@ -13,7 +13,7 @@
 
 package cninswrapper
 
-import "github.com/containernetworking/cni/pkg/ns"
+import "github.com/containernetworking/plugins/pkg/ns"
```

Comment:
Have all these packages moved under plugins directory now ? Looking at this PR in isolation is not giving me that insight. Can we still continue to use cni directory packages ?

Reply:
The upstream package was moved in [v0.6.0 back in August 2017](https://github.com/containernetworking/cni/releases/tag/v0.6.0), so we have to import the files from where they are now.

Reply:
Thanks for clarification. Lets run integration changes post this at least once again before the release.